blueprints/eslint-config-simplabs/index.js

```diff
@@ -111,6 +115,10 @@ module.exports = {
     return 'ember-cli-mocha' in this.project.dependencies();
   },
 
+  _hasEmberMocha() {
+    return 'ember-mocha' in this.project.dependencies();
```

Comment:
Should we also have a test that covers this case? (similar to https://github.com/simplabs/eslint-config-simplabs/blob/53805a9c7b3c84c304bdfc1a0df059eb351c3986/node-tests/blueprints/eslint-config-simplabs-test.js#L52)